hammr/commands/quota/quota.py

```diff
@@ -62,7 +62,11 @@ def do_list(self, args):
                     if quota.limit != -1:
                         nb = float(quota.nb)
```

Comment:
Your new code could be extracted in a function, something like compute_disk_value_when_quota_reached. And this new function could also have unit tests :)